samples/AzureKeyVault/apiDefinition.swagger.json

```diff
@@ -0,0 +1,872 @@
+            "required": true,
+            "type": "string"
+          },
+          {
+            "default": "7.0",
+            "in": "query",
+            "name": "api-version",
+            "required": true,
+            "type": "string",
+            "x-ms-visibility": "internal"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "properties": {
+                "attributes": {
+                  "description": "attributes",
+                  "properties": {
+                    "created": {
+                      "description": "created",
+                      "format": "int32",
+                      "type": "integer"
+                    },
+                    "enabled": {
+                      "description": "enabled",
+                      "type": "boolean"
+                    },
+                    "recoverylevel": {
+                      "description": "recoverylevel",
+                      "type": "string"
+                    },
+                    "updated": {
+                      "description": "updated",
+                      "format": "int32",
+                      "type": "integer"
+                    }
+                  },
+                  "type": "object"
+                },
+                "id": {
+                  "description": "id",
+                  "type": "string"
+                },
+                "value": {
+                  "description": "value",
+                  "format": "byte",
+                  "type": "string"
+                }
+              },
+              "type": "object"
+            }
+          }
+        },
+        "summary": "Get secret"
+      },
+      "put": {
+        "description": "Sets a secret in a specified key vault. This operation adds a secret to the Azure Key Vault. If the named secret already exists, Azure Key Vault creates a new version of that secret. This operation requires the secrets/set permission. For details, see: https://docs.microsoft.com/rest/api/keyvault/setsecret/setsecret.",
+        "operationId": "SetSecret",
+        "parameters": [
+          {
+            "in": "path",
+            "name": "secret-name",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "default": "7.0",
+            "in": "query",
+            "name": "api-version",
+            "required": true,
+            "type": "string",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "required": true,
+            "schema": {
+              "properties": {
+                "attributes": {
+                  "description": "attributes",
+                  "properties": {
+                    "enabled": {
+                      "description": "Determines whether the object is enabled.",
+                      "enum": [
```

Comment:
This is another bug in the portal: type is Boolean, but enums are string. Please remove enums, it also should not pass swagger validation also.

Reply:
We should file these bugs to be fixed.